packages/dapp/src/view/Account.vue

```diff
@@ -79,6 +67,14 @@
         :tld="selectedTld.name"
         @tld-renew-succeeded="reloadDomainsAndTlds"
```

Comment:
Shouldn't this event be named `domain-renew-succeeded` instead of `tld-renew-succeeded`?